src/web/mod.rs

```diff
@@ -104,6 +111,7 @@ pub fn run(soup: Blender) -> HttpResult<Listening> {
     Server {
             handlers: router,
             host: 8080.into(),
+            global: Global::from(Box::new(Mutex::new(soup))),
```

Comment:
Actually, why does the soup need to be boxed here? Given that you never *use* `global`, it should be fine to just do `Global::from(soup)`, no?